modules/loki-rate-limit-errors.adoc

```diff
@@ -0,0 +1,71 @@
+// Module is included in the following assemblies:
+// * logging/cluster-logging-loki.adoc
+// * logging/cluster-logging-external.adoc
+
+:_content-type: PROCEDURE
+[id="loki-rate-limit-errors_{context}"]
+= Troubleshooting Loki rate limit errors
+
+If the Log Forwarder API forwards a large block of messages to Loki that exceeds the rate limit, Loki generates rate limit (`429`) errors. You can fix this issue by modifying the `LokiStack` custom resource (CR).
```

Comment:
Adding some additional text too explain this, thanks so much for the examples!

Re metrics dashboards, in general for documentation we try to avoid adding unnecessary screenshots, because this introduces more images that need to be maintained. I also don't want to expand the scope of the PR too much unnecessarily.

If you think this is something that really needs to be documented, would you mind maybe opening a new Jira to enhance the docs on this by adding dashboards and explanations for them, so that we can try to keep this PR to the scope of updating the procedure, rather than adding too much new conceptual info?